extensions/_modules/heroku.py

```diff
@@ -51,164 +46,225 @@ def __virtual__():
+
+    CLI Examples:
+    salt-call heroku.list_app_addons
+    '''
+
+    result = _query(endpoint='apps',
+               arguments=app_name+'/addons',
+               api_key=api_key)
+    log.debug('result {0}'.format(result))
+    return result
 
-        salt '*' heroku.list_rooms api_key=peWcBiMOS9HrZG15peWcBiMOS9HrZG15
+def list_app_attachments(app_name, api_key = None):
     '''
-    foo = _query(endpoint='apps',
-                 api_url=api_url,
-                 api_key=api_key)
-    log.debug('foo {0}'.format(foo))
-    return foo
+    List app add-on attachments. This uses the following heroku call:
+    https://devcenter.heroku.com/articles/platform-api-reference#add-on-attachment-list
 
-def app_info(app_name,
-             api_url=None,
-             api_key=None,
-             endpoint=None):
+    CLI Examples:
+    salt-call heroku.list_app_attachments
+    '''
 
-    foo = _query(endpoint='apps',
-               arguments=app_name,
-               api_url=api_url,
+    result = _query(endpoint='apps',
+               arguments=app_name+'/addon-attachments',
                api_key=api_key)
-    log.debug('foo {0}'.format(foo))
-    return foo
+    log.debug('result {0}'.format(result))
+    return result
+
+def list_app_buildpacks(app_name, api_key = None):
+    '''
+    List an app's existing buildpack installations. This uses the following heroku call:
+    https://devcenter.heroku.com/articles/platform-api-reference#buildpack-installations-list
 
-def list_buildpacks(app_name,
-             api_url=None,
-             api_key=None,
-             endpoint=None):
+    CLI Examples:
+    salt-call heroku.list_app_buildpacks
+    '''
 
-    foo = _query(endpoint='apps',
+    result = _query(endpoint='apps',
                arguments=app_name+'/buildpack-installations',
-               api_url=api_url,
                api_key=api_key)
-    log.debug('foo {0}'.format(foo))
-    return foo
+    log.debug('result {0}'.format(result))
+    return result
 
-def list_config_vars(app_name,
-             api_url=None,
-             api_key=None,
-             endpoint=None):
+def list_app_config_vars(app_name, api_key = None):
+    '''
+    List config vars for specified app. This uses the following heroku call:
+    https://devcenter.heroku.com/articles/platform-api-reference#config-vars
 
-    foo = _query(endpoint='apps',
+    CLI Examples:
+    salt-call heroku.list_app_config_vars
+    '''
+    result = _query(endpoint='apps',
                arguments=app_name+'/config-vars',
-               api_url=api_url,
                api_key=api_key)
-    log.debug('foo {0}'.format(foo))
-    return foo
+    log.debug('result {0}'.format(result))
+    return result
 
-def set_config_vars(app_name,
-             parameters,
-             api_url=None,
-             api_key=None,
-             endpoint=None):
+def update_app_config_vars(app_name, data, api_key = None):
```

Comment:
nitpick: I think that the `data` parameter is a bit too generic. Something like `vars` or `configs` might be more meaningful given the name of the method.